airflow/utils/db.py

```diff
@@ -130,6 +130,10 @@ def initdb():
             schema='hive', port=3400))
     merge_conn(
         models.Connection(
+            conn_id='google_cloud_default', conn_type='google_cloud_platform',
```

Comment:
Ah. This was needed for the unit tests which start from the empty db, and it would throw when initializing the gcp_cloudml_hook without having a value for `google_cloud_default` (and the tests would be skipped). Not sure if the `extra` args would be useful though.